src/ui/dialogs/spring_pane.rb

```diff
@@ -207,7 +207,7 @@ def play_animation
   end
 
   def toggle_animation
-    simulate unless @simulation_data
+    simulate
```

Comment:
Is this related to the PR or left by accident?

Reply:
Yep I wanted to include this fix here on purpose, but probably should have communicated better in the description :-)